monocle/db/queries.py

```diff
@@ -641,3 +645,18 @@ def changes_and_events(es, index, repository_fullname, params):
     data = run_query(es, index, body)
     changes = [r['_source'] for r in data['hits']['hits']]
     return {'items': changes, 'total': data['hits']['total']}
+
+
+def abandoned_changes(es, index, repository_fullname, params):
```

Comment:
You could keep the same naming rule: last_abandoned_changes ?

Reply:
done